analysis_options.yaml

```diff
@@ -0,0 +1,7 @@
+include: package:very_good_analysis/analysis_options.3.1.0.yaml
```

Comment:
**note**: just as a note, check [strict type checks](https://dart.dev/guides/language/analysis-options#enabling-additional-type-checks) 🙂 

We usually use them!